mypy/stubtest.py

```diff
@@ -1019,6 +1020,37 @@ def verify_var(
             yield Error(
                 object_path, f"variable differs from runtime type {runtime_type}", stub, runtime
             )
+            return
+
+    # If the stub has a default value and it's not an enum,
+    # attempt to verify that the default value matches the runtime
+    if isinstance(runtime, enum.Enum) or not stub.has_explicit_value:
+        return
+    stub_type = mypy.types.get_proper_type(stub.type)
+    if not isinstance(stub_type, mypy.types.Instance):
+        return
+
+    node_for_stub_value = stub_type.last_known_value
+    if node_for_stub_value is None:
+        return
+
+    stub_value: object
+    if isinstance(node_for_stub_value.value, (int, bool, float)):
+        stub_value = node_for_stub_value.value
+    elif isinstance(node_for_stub_value.value, str):
+        fallback = node_for_stub_value.fallback.type
+        if fallback.fullname == "builtins.str":
+            stub_value = node_for_stub_value.value
+        elif fallback.fullname == "builtins.bytes":
+            stub_value = bytes(node_for_stub_value.value, "utf-8")
```

Comment:
I don't know about these codecs to decide whether this is an actual problem, but FWIW mypyc converts `BytesExpr.value` into `bytes` [in a bit more involved way](https://github.com/python/mypy/blob/284142d6e29d4588107f5c68f95c43986a2e26e5/mypyc/irbuild/util.py#LL182-L189C86).